scripts/debug_pagination.py

```
@@ -0,0 +1,123 @@
+"""
+import sys
+
+import requests
+
+API_BASE_URL = "https://bsmarker.utia.cas.cz/api/v1"
+USERNAME = "remisvoj@cvut.cz"
+PASSWORD = "bsmarker20250903"  # pragma: allowlist secret
+
+
+def login(username: str, password: str) -> str:
+    response = requests.post(
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)